modules/writing_observer/writing_observer/aggregator.py

```
@@ -178,7 +182,14 @@ async def merge_with_student_data(writing_data, student_data):
     return writing_data
 
 
-import writing_observer.awe_nlp
```

What is the purpose of this line?

I'm not sure I'm reading this correctly, but if I am, this is very bad form. 

If we expect to find AWE Workbench and cannot find it, we want to fail early. A failure on startup allows us to fix the bug on a deployed server. 

A silent failure will mean we're delivering bad NLP to teachers. 

What I generally do is along the lines of:

`if [can't find module]: print([Error message] exit()`

The error message contains instructions for:

1. Installing the module
2. Enabling the stub from the config file